apps/web/pages/signup.tsx

```diff
@@ -106,106 +202,203 @@ export default function Signup({ prepopulateFormValues, token, orgSlug, orgAutoA
-        <div className="sm:mx-auto sm:w-full sm:max-w-md">
-          <h2 className="font-cal text-emphasis text-center text-3xl font-extrabold">
-            {t("create_your_account")}
-          </h2>
-        </div>
-        <div className="mt-8 sm:mx-auto sm:w-full sm:max-w-md">
-          <div className="bg-default mx-2 p-6 shadow sm:rounded-lg lg:p-8">
-            <FormProvider {...methods}>
-              <form
-                onSubmit={(event) => {
-                  event.preventDefault();
-                  event.stopPropagation();
-
-                  if (methods.formState?.errors?.apiError) {
-                    methods.clearErrors("apiError");
+        <div className="flex w-full flex-col px-4 py-6 md:px-16 2xl:px-28">
+          {/* Header */}
+          {errors.apiError && (
+            <Alert severity="error" message={errors.apiError?.message} data-testid="signup-error-message" />
+          )}
+          <div className="flex flex-col gap-1">
+            <h1 className="font-cal text-[28px] ">
+              {IS_CALCOM ? t("create_your_calcom_account") : t("create_your_account")}
+            </h1>
+            {IS_CALCOM ? (
+              <p className="text-subtle text-base font-medium leading-none">{t("cal_signup_description")}</p>
+            ) : (
+              <p className="text-subtle text-base font-medium leading-none">
+                {t("calcom_explained", {
+                  appName: APP_NAME,
+                })}
+              </p>
+            )}
+          </div>
+          {/* Form Container */}
+          <div className="mt-10">
+            <Form
+              className="flex flex-col gap-4"
+              form={formMethods}
+              handleSubmit={async (values) => {
+                await signUp(values);
+              }}>
+              {/* Username */}
+              <UsernameField
+                label={t("username")}
+                username={watch("username")}
+                premium={premiumUsername}
+                usernameTaken={usernameTaken}
+                setUsernameTaken={(value) => setUsernameTaken(value)}
+                data-testid="signup-usernamefield"
+                setPremium={(value) => setPremiumUsername(value)}
+                addOnLeading={
+                  orgSlug
+                    ? getOrgFullDomain(orgSlug, { protocol: true })
+                    : `${process.env.NEXT_PUBLIC_WEBSITE_URL}/`
+                }
+              />
+              {/* Email */}
+              <TextField
+                {...register("email")}
+                label={t("email")}
+                type="email"
+                data-testid="signup-emailfield"
+              />
+
+              {/* Password */}
+              <PasswordField
+                data-testid="signup-passwordfield"
+                label={t("password")}
+                {...register("password")}
+                hintErrors={["caplow", "min", "num"]}
```

Comment:
This is flagging initially right and when password entered it doesn't change.

![image](https://github.com/calcom/cal.com/assets/4461358/e6043c28-e56c-4bb4-b8e8-5478809040ce)

Reply:
This works like this, when google autocompletes it marks everything green but when deleting input it doesn't return to original state.

Reply:
Not related to this PR - will tackle in a follow up.

Reply:
https://github.com/calcom/cal.com/issues/11486 Created issue for a follow up

Reply:
> Not related to this PR - will tackle in a follow up.

@sean-brydon I saw you checked Loom I recorded but will post info also here.

The password validation checks work on main locally and also in prod. See Loom [here](https://www.loom.com/share/786f320e03d04aa991b9e4c40e27a9f8). So we need to find what fix is needed in this PR.

Reply:
> > Not related to this PR - will tackle in a follow up.
> 
> @sean-brydon I saw you checked Loom I recorded but will post info also here.
> 
> The password validation checks work on main locally and also in prod. See Loom [here](https://www.loom.com/share/786f320e03d04aa991b9e4c40e27a9f8). So we need to find what fix is needed in this PR.

Super weird - i copied and pasted it :P Will see if i can see anything obvious

Reply:
@sean-brydon I pushed commit [3e8470b](https://github.com/calcom/cal.com/pull/11421/commits/3e8470b8dd714f98d0f686f7830f01125c23fec7) to this branch that fixes the password hints not being dynamic. In [this commit](https://github.com/calcom/cal.com/pull/11421/commits/dc277a0afa2f70f6fdeb8871657a933032dd1442) `mode: "onChange"` was deleted from the `useForm` hook and it was not re-added later on.